package.json

```diff
@@ -90,6 +90,7 @@
     "redux": "^3.7.2",
     "redux-api-middleware": "^2.3.0",
     "redux-thunk": "^2.3.0",
-    "select2": "^4.0.6-rc.1"
+    "select2": "^4.0.6-rc.1",
+    "whatwg-fetch": "^3.0.0"
```

Comment:
When I `npm install` my package-lock was slightly different.. did you commit the latest one after installing all the new dependencies?